README.markdown

```diff
@@ -365,7 +370,12 @@ There are a few known issues on Windows platforms:
 
 * MSI installation failures do not produce any error. If the install fails, puppet_agent continues to be applied to the agent. If this happens, you'll need to examine the MSI log file to determine the failure's cause. You can find the location of the log file in the debug output from either a puppet apply or an agent run; the log file name follows the pattern `puppet-<timestamp>-installer.log`.
 
-* Upgrading on most \*NIX platforms (Linux, AIX, Solaris 11) will end the run after the puppet-agent upgrade finishes. This is to avoid unexpected behavior if already loaded Ruby code happens to interact with newer code that came with the upgrade, or viceversa. If run as a daemon, Puppet will automatically start a new agent run after the upgrade finishes.
+* If the upgrade is from Puppet 5 prior to 5.5.17 or Puppet 6 prior to 6.8.0 to newer version and `msi_move_locked_files` is set to `true`, Puppet can get into a state where `puppet --versions` reports the older version (5.5.16) while the package reported by windows is the new version(5.5.17). To recoved from this case `ADDLOCAL=ALL` must be added to install_options
+``` puppet
+  install_options => ['REINSTALLMODE="maus"', 'ADDLOCAL=ALL']
```

Comment:
Think this is `amus` not `maus`. This is also on line 283 and 298.

Actually it should be order-independent (https://docs.microsoft.com/en-us/windows/win32/msi/reinstallmode), but in our code we hardcode `amus` so it may be good to have the same value in the documentation.